Source/ARTRealtimePresence.m

```diff
@@ -153,6 +154,7 @@ @implementation ARTRealtimePresenceInternal {
     __weak ARTRealtimeChannelInternal *_channel; // weak because channel owns self
     dispatch_queue_t _userQueue;
     NSMutableArray<ARTQueuedMessage *> *_pendingPresence;
+    ARTEventEmitter<ARTEvent *, ARTPresenceMessage *> *_presenceEventEmitter;
```

Comment:
Is the word "presence" needed here still?

Reply:
646f555a1e037aaa2a43ebe440343e3ff1396075